core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html

```diff
@@ -227,4 +288,75 @@
     color: #fff;
     padding: 1px 5px;
   }
+  .oppia-contributor-home {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 56px;
+    z-index: 3;
+  }
+  .headroom--unpinned {
+    transform: translateY(0%);
+  }
```

Comment:
Is this being used anywhere? Can you take a pass on all the CSS definitions and remove any that are not being used?

Reply:
This headroom class was added globally to hide the headers on scroll. I used this in order to prevent the effect of headroom. https://wicky.nillia.ms/headroom.js/

Reply:
Ah, I see thanks.